.gitignore

```diff
@@ -10,6 +10,7 @@ lerna-debug.log*
 build/
 
 node_modules
+coverage
```

Comment:
Not necessary. Check on the other P.Rs. I left a similar comment.

Reply:
@shyakadavis @SergeInGithub If there is a PR which was merged having this fix, a rebase will solve this and it won't be tracked as a new change.

Reply:
Indeed.

I was pointing out that he didn't need to include it once again.

Reply:
@shyakadavis @mucyoalexaxel Thanks for noting this out, for some reason i had missed it, but it's all up fixed now.